Source/GSXib5KeyedUnarchiver.m

```diff
@@ -3318,6 +3321,14 @@ - (id) decodeObjectForKey: (NSString *)key
           // New xib stores values as attributes...
           object = [currentElement attributeForKey: key];
         }
+      else if ([@"NSCollectionViewBackgroundColor" isEqualToString: key])
```

Comment:
NSCollectionViewBackgroundColor (line 3324), if it is TRULY mapped to one attribute named primaryBackgroundColor, could be added to the XmlKeyMapTable to be translated.

Of course, unless the processing sequence has been broken somewhere.

Reply:
I don't think that's the problem in this case.